packages/web/pages/docs/rest-api.mdx

```diff
@@ -72,12 +72,12 @@ is included in the request.
       headers: {
         "Content-Type": "application/json",
       },
-      body: {
+      body: JSON.stringify({
         "templateName": "myTemplate",
         "props": {
           "name": "Alex"
         },
-      }
+      }),
```

Comment:
Added this trailing comma to stylistically match the `/api/sendMail` example. Easy to remove if you don't want this in the docs.